concepts/05 Widgets/SelectBox/00 Getting Started with SelectBox/25 Group Data.md

```diff
@@ -0,0 +1,147 @@
+            Name: 'Apple',
+            Category: 'Fruits',
+        }, {
+            ID: 4,
+            Name: 'Tomato',
+            Category: 'Vegetables',
+        }, {
+            ID: 5,
+            Name: 'Apricot',
+            Category: 'Fruits',
+        }]
+
+        const dataSource = new DevExpress.data.DataSource({
+            store: data,
+            type: 'array',
+            key: "ID",
+            group: "Category"
+        });
+
+        $("#selectBox").dxSelectBox({
+            // ...
+             grouped: true
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-select-box ...
+        [grouped]="true">
+    </dx-select-box>
+
+    <!-- tab: app.component.ts -->
+    // ...
+    import DataSource from 'devextreme/data/data_source';
+
+    // ...
+    export class AppComponent {
+        data: Item[];
+        dataSource: DataSource;
+
+        constructor(service: AppService) {
+            this.data = service.getItems();
+            this.dataSource = new DataSource({
+                store: {
+                    data: this.data,
+                    type: 'array',
+                    key: 'ID'
+                },
+                group: "Category"
+            });
+        }
+        // ...
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxSelectBox ...
+            :grouped="true"
+        />
+    </template>
+
+    <script>
+    // ...
+    import DataSource from 'devextreme/data/data_source';
+
+    export default {
+        components: {
+            DxSelectBox
+        },
+        data() {
+            return {
+                dataSource: new DataSource({
+                    store: {
+                        data: data,
+                        type: 'array',
+                        key: 'ID',
+                    },
+                    group: "Category"
+                })
+            }
+        },
+        // ...
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    // ...
+    import DataSource from 'devextreme/data/data_source';
+
+    const dataSource = new DataSource({
+        store: {
+            data: data,
+            type: 'array',
+            key: 'ID'
+        },
+        group: "Category"
+    })
+
+    class App extends React.Component {
+        // ...
+
+        render() {
+            return (
+                <SelectBox ...
+                    grouped={true}
+                />
+            );        
+        }
+
+    }
+
+    export default App;
+    
+---
+
+If your data is already grouped, ensure each group has the **key** and **items** fields as shown in [this article](/Documentation/Guide/Widgets/SelectBox/Grouping/In_the_Data_Source/). If the fields are named differently, implement the **DataSource**'s [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** + **items** field mappings.
+
+Run the code and ensure the widget's data is grouped.
```

Comment:
Run code and ensure the widget's data is grouped.

is it and ensure or to ensure?

Reply:
ensure.